docs/modules/index.ts.md

```diff
@@ -79,51 +64,48 @@ Tests the `Apply` laws
 export function apply<F extends URIS3>(
   F: Apply3<F>
 ): <U, L>(
-  lift: <A>(a: fc.Arbitrary<A>) => fc.Arbitrary<Type3<F, U, L, A>>,
-  liftSetoid: <A>(Sa: Setoid<A>) => Setoid<Type3<F, U, L, A>>
-) => void
-export function apply<F extends URIS3, U, L>(
-  F: Apply3C<F, U, L>
```

Comment:
Are all the 3C gone in v2? I guess because they're now unused internally?

Reply:
> Are all the 3C gone in v2?

y